source/community.applications/usr/local/emhttp/plugins/community.applications/skins/Narrow/styles/community.applications.css

```
@@ -3307,56 +3317,61 @@ input[type="button"] {
+	body.ca_searchModalOpen.ca_awesomplete_open #searchFilter .awesomplete,
+	body.ca_searchModalOpen.ca_awesomplete_open #searchFilter .awesomplete *,
+	body.ca_searchModalOpen.ca_awesomplete_open .searchButton,
+	body.ca_searchModalOpen.ca_awesomplete_open .caSearchModalIconControls,
+	body.ca_searchModalOpen.ca_awesomplete_open .caSearchModalIconControls * {
 		pointer-events: auto;
 	}
 
-	.awesomplete .visually-hidden {
+	body.ca_searchModalOpen #searchFilter .awesomplete .visually-hidden {
 		position: absolute;
 		clip: rect(0, 0, 0, 0);
```

Stylelint (17.11.1)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)